src/control/Control.js

```diff
@@ -104,8 +104,9 @@ export const Control = Class.extend({
 	},
 
 	_refocusOnMap(e) {
-		// if map exists and event is not a keyboard event
-		if (this._map && e && e.screenX > 0 && e.screenY > 0) {
+		// We exclude keyboard-click event to keep the focus on the control for accessibility.
+		// The position of keyboard-click events are x=0 and y=0.
+		if (this._map && e && !(e.screenX === 0 && e.screenY === 0)) {

```

Comment:
I changed this check because `e.screenX > 0 && e.screenY > 0` didn't worked on my second screen because `screenX` was `-1700`